cookbook/service_container/event_listener.rst

```diff
@@ -1,27 +1,36 @@
 .. index::
    single: Events; Create listener
+   single: Create subscriber
 
-How to Create an Event Listener
-===============================
+How to Create Event Listeners and Subscribers
+=============================================
 
-Symfony has various events and hooks that can be used to trigger custom
-behavior in your application. Those events are thrown by the HttpKernel
-component and can be viewed in the :class:`Symfony\\Component\\HttpKernel\\KernelEvents` class.
+Symfony has various events and hooks that can be used to perform custom
+actions in your application. Those events are triggered by the HttpKernel
+component and they are defined in the :class:`Symfony\\Component\\HttpKernel\\KernelEvents`
+class.
```

Comment:
Probably, adding a short note after this paragraph that you can dispatch your own events using the core event dispatcher and that there are lots of more events provided by third-party libraries, could be sufficient (we can link to related articles).